core/src/world/place/building/religious/shrine.rs

```diff
@@ -17,7 +17,7 @@ fn name(rng: &mut impl Rng) -> String {
         }
         9 => {
             let (animal, s) = pluralize(word::animal(rng));
-            format!("{} of the {} {}{}", place(rng), number(rng), animal, s)
+            format!("{} of the {} {}{}", place(rng), word::number(rng), animal, s)
```

Comment:
I appreciate the effort to reduce repetition in existing code, but especially with the expanded list of numbers, things start to get a bit silly. Examples from the tests:

- Thirty Four Printers
- One Hundred and Twenty Four Kegs
- One Hundred and Twenty Two Beggars
- One Hundred and Twenty Eight Helmets Concert Hall
- Shrine of the One Hundred and Twenty Four Beavers
- Altar of the One Hundred and Twenty Two Tigers

You might encounter large but oddly specific numbers every now and then, but the probability should be much lower. Humans tend to like their threes, fives, and so forth.